lib/apis/moray.js

```diff
@@ -367,11 +369,9 @@ Moray.prototype._vmsListParams = function (params, cb) {
     }
 
     if (params.create_timestamp) {
-        var ts = params.create_timestamp;
-        var input = isNaN(Number(ts)) ? ts : Number(ts);
-        var tts = new Date(input).getTime();
-        if (!isNaN(tts)) {
-            filter.push(sprintf(PARAM_FILTER, 'create_timestamp', tts));
+        var ts = common.convertDateStringToNumeric(params.create_timestamp);
```

Comment:
`params.create_timestamp` is already validated [at another layer, before this code is called](https://github.com/joyent/sdc-vmapi/blob/master/lib/common/validation.js#L1289).